Update Groups snippet on docs (#137)
amplitude/amplitude-dev-center: docs/data/sdks/go/ampli.md

    },
)
```

> ### Groups

Reviewer: ```suggestion

### Groups
```

> Groups are not yet supported with the Amplitude Go SDK or Go Ampli Wrapper. If you'd like Groups to be added as a feature, please submit a feature request via the widget in your dashboard or through a ticket at this [link](https://support.amplitude.com).

Reviewer: ```suggestion
The Amplitude Go SDK and Go Ampli Wrapper don't support Groups. If you're interested in this feature, submit a feature request through the widget on the Amplitude dashboard, or through a [support ticket](https://support.amplitude.com).
```
